### PR TITLE
Add Danish translation

### PR DESCRIPTION
**Pull request by OEHC**

### CODE REVIEW

File: custom_components/husqvarna_automower/translations/da.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                "description": "Kamera indstillinger",
+                "title": "Husqvarna Automower indstillinger"
+            }
+        }
+    },
+    "system_health": {
+        "info": {
+            "can_reach_server": "Kan nå Husqvarna Token server"
+        }
+    }
```
Comment (Thomas55555) [inline custom_components/husqvarna_automower/translations/da.json:62]: Can you add a translation for this?
https://github.com/Thomas55555/husqvarna_automower/blob/f8b8a28f01a331455016dd9321cae2589ed23eba/custom_components/husqvarna_automower/translations/en.json#L42-L43
 I've just changed the strings.json and it's new now.

**Review by Thomas55555 — CHANGES_REQUESTED** [review]
Can you update the translation, like in the review?

File: custom_components/husqvarna_automower/translations/da.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                    "password": "Password",
+                    "credentials": "Husk brugernavn og password (muliggører at denne komponent kan hente en ny token hvis den ikke længere kan fornyes)"
+                }
+            },
+            "user": {
+                "menu_options": {
+                    "password": "Log ind med API key, brugernavn og password",
+                    "oauth2": "Log ind med Authorization Code Grant (yaml nødvendig)"
+                }
+            },
```
[the remark this answers is not in the document]
Comment (OEHC) [reply custom_components/husqvarna_automower/translations/da.json:18]: Fixed in da7ce26

File: custom_components/husqvarna_automower/translations/da.json (hunk 1 above)
Comment (OEHC) [reply custom_components/husqvarna_automower/translations/da.json:62]: Fixed in da7ce26

**Review by Thomas55555 — APPROVED** [review]

**Pull request merged**